servicetalk-grpc-api/src/main/java/io/servicetalk/grpc/api/GrpcUtils.java

```diff
@@ -241,14 +239,14 @@ private static void ensureGrpcStatusReceived(final HttpHeaders headers) {
         }
     }
 
-    static GrpcMessageEncoding readGrpcMessageEncoding(final HttpMetaData httpMetaData,
-                                                       final Set<GrpcMessageEncoding> allowedEncodings) {
+    static ContentCodec readGrpcMessageEncoding(final HttpMetaData httpMetaData,
```

Comment:
Sad that we need to duplicate these logic in HTTP and gRPC :(
Hope we can have a better way to share it in the future.

Reply:
Agreed offline to move these in a separate module in a separate PR